### ISSUE TITLE
Error adding a registered user to a circle in Contacts checking to accept invitation

### ISSUE THREAD

**Issue by ChemBioScripting** [nextcloud/android#10002]
### Steps to reproduce
1. Create a circle on Nextcloud contacts
2. Check "members need to accept invitation"
3. Add a registered member to the circle
4. The user will receive a notification into the Nextcloud app asking to accept or reject the invitation
5. The click on accept generates the following error

### Expected behaviour
- Tell us what should happen
The user should be added to the circle

### Actual behaviour
- Tell us what happens
An error happen without adding the user to the circle

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:
11

Device model: 
Brand: OPPO
Device: OP4F39L1
Model: CPH2195

Stock or customized system:
Stock

Nextcloud app version:
3.19.1

Nextcloud server version:
23.0.2

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

************ CAUSE OF ERROR ************
```
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done([AsyncTask.java:415](http://asynctask.java:415/))
	at java.util.concurrent.FutureTask.finishCompletion([FutureTask.java:383](http://futuretask.java:383/))
	at java.util.concurrent.FutureTask.setException([FutureTask.java:252](http://futuretask.java:252/))
	at [java.util.concurrent.FutureTask.run](http://java.util.concurrent.futuretask.run/)([FutureTask.java:271](http://futuretask.java:271/))
	at android.os.AsyncTask$SerialExecutor$[1.run](http://1.run/)([AsyncTask.java:305](http://asynctask.java:305/))
	at java.util.concurrent.ThreadPoolExecutor.runWorker([ThreadPoolExecutor.java:1167](http://threadpoolexecutor.java:1167/))
	at java.util.concurrent.ThreadPoolExecutor$[Worker.run](http://worker.run/)([ThreadPoolExecutor.java:641](http://threadpoolexecutor.java:641/))
	at [java.lang.Thread.run](http://java.lang.thread.run/)([Thread.java:923](http://thread.java:923/))
Caused by: java.lang.IllegalArgumentException: host parameter is null
	at org.apache.commons.httpclient.HttpConnection.<init>([HttpConnection.java:206](http://httpconnection.java:206/))
	at org.apache.commons.httpclient.HttpConnection.<init>([HttpConnection.java:155](http://httpconnection.java:155/))
	at org.apache.commons.httpclient.MultiThreadedHttpConnectionManager$HttpConnectionWithReference.<init>([MultiThreadedHttpConnectionManager.java:1145](http://multithreadedhttpconnectionmanager.java:1145/))
	at org.apache.commons.httpclient.MultiThreadedHttpConnectionManager$ConnectionPool.createConnection([MultiThreadedHttpConnectionManager.java:762](http://multithreadedhttpconnectionmanager.java:762/))
	at org.apache.commons.httpclient.MultiThreadedHttpConnectionManager.doGetConnection([MultiThreadedHttpConnectionManager.java:476](http://multithreadedhttpconnectionmanager.java:476/))
	at org.apache.commons.httpclient.MultiThreadedHttpConnectionManager.getConnectionWithTimeout([MultiThreadedHttpConnectionManager.java:416](http://multithreadedhttpconnectionmanager.java:416/))
	at org.apache.commons.httpclient.HttpMethodDirector.executeMethod([HttpMethodDirector.java:153](http://httpmethoddirector.java:153/))
	at org.apache.commons.httpclient.HttpClient.executeMethod([HttpClient.java:397](http://httpclient.java:397/))
	at org.apache.commons.httpclient.HttpClient.executeMethod([HttpClient.java:323](http://httpclient.java:323/))
	at [com.owncloud.android](http://com.owncloud.android/).lib.common.OwnCloudClient.executeMethod([OwnCloudClient.java:205](http://owncloudclient.java:205/))
	at [com.owncloud.android](http://com.owncloud.android/).ui.asynctasks.NotificationExecuteActionTask.doInBackground([NotificationExecuteActionTask.java:70](http://notificationexecuteactiontask.java:70/))
	at [com.owncloud.android](http://com.owncloud.android/).ui.asynctasks.NotificationExecuteActionTask.doInBackground([NotificationExecuteActionTask.java:22](http://notificationexecuteactiontask.java:22/))
	at android.os.AsyncTask$3.call([AsyncTask.java:394](http://asynctask.java:394/))
	at [java.util.concurrent.FutureTask.run](http://java.util.concurrent.futuretask.run/)([FutureTask.java:266](http://futuretask.java:266/))
	... 4 more
```
************ APP INFORMATION ************
ID: com.nextcloud.client
Version: [30190190](tel:30190190)
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: OPPO
Device: OP4F39L1
Model: CPH2195
Id: RKQ[1.201217.002](tel:1201217002)
Product: CPH2195EEA

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: 1646051176501

**Comment by samyberkane04** [comment]
@ChemBioScripting 
hello can i work on this issue please ?
thank you.

**Comment by joshtrichards** [comment]
The error suggests the URL in the notification for the accept/reject was strange. Some configuration situations might cause this (like misbehaving reverse proxies - i.e. #10664). There was also a relevant bug in Circles around that time (fixed in nextcloud/circles#735).

Can you reproduce this behavior still today with a newly generated invitation?
